crates/solvers/src/boundary/baseline.rs

```diff
@@ -25,7 +26,7 @@ impl<'a> Solver<'a> {
         weth: &eth::WethAddress,
         base_tokens: &HashSet<eth::TokenAddress>,
         liquidity: &'a [liquidity::Liquidity],
-        uni_v3_quoter_v2: Option<Arc<contracts::UniswapV3QuoterV2>>,
+        uni_v3_quoter_v2: Option<Arc<contracts::alloy::UniswapV3QuoterV2::Instance>>,
```

Comment:
Let's keep the used names shorter please. With the 2 extra indirection `alloy` has this is too long.